### PR TITLE
Updated Chart and DataGrid examples to use FDC3 standard

### PR DESCRIPTION
**Pull request by lilla28**
Updated example projects to use FDC3 standard.
image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
         "@angular/platform-browser": "^15.1.0",
         "@angular/platform-browser-dynamic": "^15.1.0",
         "@angular/router": "^15.1.0",
-        "@morgan-stanley/composeui-messaging-client": "*",
+        "@finos/fdc3": "2.0",
```
Comment (kruplm) [inline package-lock.json:108]: That's interesting. The npm package has patch versions, like you use it on the chart.
https://www.npmjs.com/package/@finos/fdc3
I don't know how it hasn't failed :D 
But just to be sure please include the patch version for 2.0.3

File: src/shell/js/composeui-fdc3/package.json [hunk 2]
```diff
@@ -3,8 +3,9 @@
   "version": "0.1.0",
   "description": "FDC3 DesktopAgent implementation for Compose UI",
   "type": "module",
-  "main": "dist/index.js",
-  "module": "dist/index.js",
+  "main": "dist/esm/index.js",
```
Comment (kruplm) [inline src/shell/js/composeui-fdc3/package.json:6]: Please revert this. I don't export ESM from this package.

**Review by kruplm — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (lilla28) [reply package-lock.json:108]: Sure will replace it with the patched version!

**Review by kruplm — DISMISSED** [review]

File: examples/js-datagrid/src/app/main-view/services/mock-data.service.ts [hunk 3]
```diff
@@ -13,61 +12,59 @@ export class MockDataService{
...
 
-  private async checkMessageRouterConnection(){
-    if(!this.connected){
-      await this.client.connect();
-      this.connected = true;
+      await window.fdc3.broadcast(context);
     }
   }
+
+  // public async requestSymbolData(topic: string) : Promise<void>{
```
Comment (ZKRobi) [inline examples/js-datagrid/src/app/main-view/services/mock-data.service.ts:64]: Don't leave commented code

Comment (lilla28) [reply examples/js-datagrid/src/app/main-view/services/mock-data.service.ts:64]: that was on purpose as per It should (in a later state) involve DataService.
I'll remove it then!

**Review by ZKRobi — APPROVED** [review]

**Pull request merged**